Formula/b/boost.rb

```diff
@@ -1,10 +1,9 @@
 class Boost < Formula
```

Comment:
I wonder if there is any risk in patching out the versioned library on Linux. Though Arch, Fedora, etc. do keep have them, e.g. https://archlinux.org/packages/extra/x86_64/boost-libs/

Not sure what Debian is doing but libs aren't versioned while other directories/files are: https://packages.debian.org/sid/amd64/libboost-filesystem1.83-dev/filelist

---

On macOS, the libs don't include version info so revision bumps usually aren't needed https://github.com/boostorg/boost/blob/boost-1.85.0/boostcpp.jam#L203-L216

Reply:
I guess ABI incompatibility may result in rebuilds anyway: https://abi-laboratory.pro/?view=timeline&l=boost